kafka/producer/base.py

```diff
@@ -60,11 +62,13 @@ def _send_upstream(queue, client, codec, batch_time, batch_size,
             # Adjust the timeout to match the remaining period
             count -= 1
             timeout = send_at - time.time()
-            msgset[topic_partition].append((msg, key))
+            msgset[topic_partition].append(msg)
```

Comment:
I think it was there because of the version we were using ``0.9.3``.  It changed with the current master.